test/run_pass/test_mqtt.js

```diff
@@ -35,27 +40,37 @@ var received2 = false;
 
 var subClientOpts = {
   clientId: 'iotjs-mqtt-test-sub',
-  host: 'test.mosquitto.org',
+  host: 'localhost',
```

Comment:
As far as I know it is possible to redirect test.mosquitto.org to localhost, so there is no need mosquitto server to run test locally.